vexim/adminuseraddsubmit.php

```diff
@@ -158,7 +158,7 @@
       mail("{$_POST['localpart']}@{$_SESSION['domain']}",
         vexim_encode_header(sprintf(_("Welcome %s!"), $_POST['realname'])),
         "$welcome_message",
-        "From: {$_SESSION['localpart']}@{$_SESSION['domain']}\r\nMIME-Version: 1.0\r\nContent-Type: text/plain; charset=utf-8\r\nContent-Transfer-Encoding: 8bit\r\n");
```

Comment:
I mean, if `$_SESSION[`localpart`] is undefined here, we should either define it somewhere (upon successful login?), or use different means to compose admin's email address (e.g. by storing her address in a session variable).